### PR TITLE
✨ Add resizable annotation mode

### PR DESCRIPTION
**Pull request by damianpumar**
Here is the first approach for resize annotation mode sides, the issue is https://github.com/argilla-io/argilla/issues/3395

https://github.com/argilla-io/argilla/assets/7398909/eb10b61b-5f25-4837-af1b-4589dee934bb

### CODE REVIEW

**Comment by dvsrepo** [comment]
this is amazing @damianpumar 🚀

**Comment by damianpumar** [comment]
> this is amazing @damianpumar 🚀

Is just the first approach, tomorrow I will present it to the team.

File: frontend/components/commons/resizable/Resizable.vue [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    return {
+      prevClientX: 0,
+      prevLeftWith: 0,
+      resizer: null,
+      leftSide: null,
+      rightSide: null,
+    };
+  },
+  mounted() {
+    this.resizer = document.getElementById("resizableBar");
```
Comment (leiyre) [inline frontend/components/commons/resizable/Resizable.vue]: Could we use $ref here instead of id?

Comment (damianpumar) [reply frontend/components/commons/resizable/Resizable.vue]: Yes, true! 
Thanks to catch it!

**Comment by davidberenstein1957** [comment]
Looking great! Two minor comments. Would it be possible to:
- keep the state of the positon of the scrollbar during a page refresh?
- also change the position when hovering on the middle line apart from the button?

**Comment by nataliaElv** [comment]
Closes #3395

**Pull request closed without merging**